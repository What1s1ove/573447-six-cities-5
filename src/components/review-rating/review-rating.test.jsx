import * as React from 'react';
import {render} from 'enzyme';
import ReviewRating from './review-rating';

describe(`ReviewRating component`, () => {
  it(`should render ReviewRating component`, () => {
    const component = render(
        <ReviewRating
          value="value"
          name="name"
          isChecked={false}
          onRatingChange={jest.fn()}
        />
    );

    expect(component).toMatchSnapshot();
  });
});
