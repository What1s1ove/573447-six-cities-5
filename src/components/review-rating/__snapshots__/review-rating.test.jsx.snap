// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewRating component should render ReviewRating component 1`] = `
Array [
  <input
    class="form__rating-input visually-hidden"
    id="value-stars"
    name="rating"
    required=""
    type="radio"
    value="value"
  />,
  <label
    class="reviews__rating-label form__rating-label"
    for="value-stars"
    title="name"
  >
    <svg
      class="form__star-image"
      height="33"
      width="37"
    >
      <use
        href="#icon-star"
      />
    </svg>
  </label>,
]
`;
